packaging/observability/microshift-observability.service

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description=MicroShift Observability
+BindsTo=microshift.service
+After=microshift.service
+AssertPathExists=/var/lib/microshift/certs/kube-apiserver-localhost-signer/observability/client.key
+AssertPathExists=/var/lib/microshift/certs/kube-apiserver-localhost-signer/observability/client.crt
```

Comment:
I'm not sure why this is necessary. We declare the current unit `After` MicroShift, so it should only start when MicroShift declares it's running, right? This means, certificates are created for sure.
What am I missing?